core-bundle/src/Command/Backup/AbstractBackupCommand.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command\Backup;
+
+use Contao\CoreBundle\Doctrine\Backup\BackupManager;
+use Contao\CoreBundle\Doctrine\Backup\Config\AbstractConfig;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+
+abstract class AbstractBackupCommand extends Command
+{
+    protected BackupManager $backupManager;
+
+    public function __construct(BackupManager $backupManager)
+    {
+        $this->backupManager = $backupManager;
+
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('file', InputArgument::OPTIONAL, 'The path to the SQL dump to process.')
+            ->addOption('ignore-tables', 'i', InputOption::VALUE_OPTIONAL, 'A comma-separated list of database tables to ignore. Defaults to the backup configuration (contao.backup.ignore_tables).')
```

Comment:
are you sure the option value is optional? Also, we might want to use the array format, such command usually work like `--ignore-table=tl_foo --ignore-table=tl_bar`

Reply:
Yes it is optional. If you don't provide anything, the config is taken into account. And I prefer the comma-separated notation because I don't want to get sore fingers from typing.